stempy/reader.h

```diff
@@ -13,25 +14,27 @@ class H5ReadWrite;
 
 namespace stempy {
 
+using Coordinates2D = std::pair<int, int>;
+using Dimensions2D = std::pair<uint32_t, uint32_t>;
```

Comment:
Lets add a comment here to explain the convention as in coordinate x,y and dimensions width, height ...

Reply:
Fixed.